coordinates/sites.json

```
@@ -569,14 +569,14 @@
         ]
     },
     "srt": {
-        "source": "SRT website & google maps",
-        "elevation": 650.0,
+        "source": "SRT single dish tools & pulsar software such as TEMPO2 and PINT. Converted through EarthLocation(4865182.7660, 791922.6890, 4035137.1740, unit=u.m).to_geodetic()",
```

```suggestion
        "source": "SRT single dish tools & pulsar software such as TEMPO2 and PINT. Converted through EarthLocation(4865182.7660, 791922.6890, 4035137.1740, unit=u.m).to_geodetic() on 2022-11-21",
```
I have no idea when or by how much the Geoid might be revised at any time in the future, or how we can get notified of it, but at least this gives us a reminder so we may update it again.